functions.php

```diff
@@ -133,3 +133,44 @@ function twentytwentyfive_pattern_categories() {
 endif;
 
 add_action( 'init', 'twentytwentyfive_pattern_categories' );
+
+add_action( 'init', 'twentytwentyfive_register_block_bindings', 9 );
+
+/**
+ * Register block bindings.
+ */
+if ( ! function_exists( 'twentytwentyfive_register_block_bindings' ) ) :
+	/**
+	 * Register the copyright block binding source.
+	 *
+	 * @since Twenty Twenty-Five 1.0
+	 * @return void
+	 */
+	function twentytwentyfive_register_block_bindings() {
+		register_block_bindings_source(
+			'twentytwentyfive/copyright',
+			array(
+				'label'              => __( 'Copyright', 'twentytwentyfive' ),
+				'get_value_callback' => 'twentytwentyfive_copyright_binding',
+			)
+		);
+	}
+	/**
+	 * Callback function for the copyright block binding source.
+	 *
+	 * @since Twenty Twenty-Five 1.0
+	 * @return string Copyright text.
+	 */
+	function twentytwentyfive_copyright_binding() {
+		/* translators: Copyright date format, see https://www.php.net/manual/datetime.format.php */
+		$date_format    = _x( 'Y', 'copyright date format', 'twentytwentyfive' );
+		$copyright_text = sprintf(
+			/* translators: 1: Copyright symbol or word, 2: Date */
+			__( '%1$s %2$s', 'twentytwentyfive' ),
```

Comment:
I believe we may want to use an escaping function here, as `wp_date` is hookable, and it could lead into some security issues.